lib/features/outline/OutlineUtil.js

```diff
@@ -0,0 +1,24 @@
+import { assign } from 'min-dash';
+
+import {
+  attr as svgAttr
+} from 'tiny-svg';
+
+
+export const DATA_OBJECT_REFERENCE_OUTLINE_PATH = 'M44.7648 11.3263L36.9892 2.64074C36.0451 1.58628 34.5651 0.988708 33.1904 0.988708H5.98667C3.22688 0.988708 0.989624 3.34892 0.989624 6.26039V55.0235C0.989624 57.9349 3.22688 60.2952 5.98667 60.2952H40.966C43.7257 60.2952 45.963 57.9349 45.963 55.0235V14.9459C45.963 13.5998 45.6407 12.3048 44.7648 11.3263Z';
```

Comment:
How did you get this path? Just curious; I think it's OK to leave it as is.

Reply:
This is the extracted path from the `svg` file

Reply:
Related to https://github.com/bpmn-io/bpmn-js/pull/1998#discussion_r1377271266, it would be great to also make it work for non-standard size. I think it would be also OK to just use the rectangular outline if we detect non-standard bounds if it's easier to implement.

Reply:
Ok so I made sure the standard shapes work with any size. For custom shapes, let's keep it for bpmn-io standard sizes and default to the general rectangular shape. We can invest further into this in the future if it makes sense.

Reply:
Sounds good to me.